EIPS/eip-223.md

```diff
@@ -0,0 +1,368 @@
+---
+eip: 223
+title: 223 Token with communication model
+description: Token with event handling and communication model
+author: Dexaran (@Dexaran)
+discussions-to: https://ethereum-magicians.org/t/erc-223-token-standard/12894
+status: Review
+type: Standards Track
+category: ERC
+created: 2017-05-03
+---
+
+## Abstract
+
+The following describes an interface for fungible tokens that supports a `tokenReceived` callback to notify contract recipients when tokens are received.
+
+## Motivation
+
+This token introduces a communication model for contracts that can be utilized to straighten the behavior of contracts that interact with such tokens. Specifically, this proposal:
+
+1. Informs receiving contracts of tokens, as opposed to [ERC-20](./eip-20.md) where the recipient of a token transfer gets no notification.
+2. Is more gas-efficient when depositing tokens to contracts.
+3. Allows for `_data` recording for financial transfers.
+
+
+## Specification
+
+
+### Methods
+
+Contracts intending to receive these tokens MUST implement `tokenReceived`.
+
+Tokens transferred to contracts not implementing `tokenReceived` as described below MUST revert.
+
```

Comment:
I don't see the `tokensReceived` hook here — maybe you misplaced the above?

Reply:
fixed

Reply:
`tokenReceived()` is described here https://github.com/Dexaran/EIPs/blob/patch-1/EIPS/eip-223.md#contract-that-is-intended-to-receive-erc-223-tokens

or what is the issue with this statement?

Reply:
Ah. It would be helpful to bring those two statements together.  Generally,  'below' means 'directly below'.

Reply:
I have moved the `tokenReceived()` paragraph directly below `transfer()` functions 

https://github.com/Dexaran/EIPs/commit/7b67c2ecfcc110c6589f1c627df65f44f9e76aeb